javacutil/src/main/java/org/checkerframework/javacutil/SystemUtil.java

```diff
@@ -225,4 +226,22 @@ public static int getJreVersion() {
         }
         return javaHome + File.separator + "lib" + File.separator + "tools.jar";
     }
+
+    /**
+     * Given an array and two values, creates an array with all the values. The two extra values are
+     * at the end of the returned array.
+     *
+     * @param <T> the type of the array elements
+     * @param array the array to extend
+     * @param elt1 the first element to add
+     * @param elt2 the second element to add
+     * @return a copy of {@code array}, with two elements added at the end
+     */
+    public static <T> T[] arrayPlusTwoElements(T[] array, T elt1, T elt2) {
```

Comment:
Is there a reason to limit this to two elements instead of making it a varargs method?